tests/client/core/test_middleware.js

```diff
@@ -68,17 +70,51 @@ describe('Prefix Middleware', () => {
     assert.deepEqual(fakeRes.set.firstCall.args, ['vary', []]);
   });
 
-  it('should render a 404 when a URL exception is found', () => {
+  it('should render 404 when a localeURL exception exists', () => {
```

Comment:
Yeah. Well, all of our routes are prepended with `/:lang/:clientApp/` so we don't have routes that look like these.

I guess if we were eventually serving them it might be weird but I don't think we'd do so with the React Router anyway, so I think it's fine.